rails-commenting.rb

```diff
@@ -7,24 +7,33 @@
 # FILE: app/controller/blog_posts_controller.rb
 
 # ---1)
+# I believe this is the model class which is automatically generated  when we generate a model and run a database migration. 
+# def index is a controller method that will allow access to all the items from the database. 
 class BlogPostsController < ApplicationController
```

Comment:
So this assignment was just looking at the line immediately below the number. for example, Just need to explain Line 12